.github/CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 ## Release 0.2.0 (development release)
 
+### New features since last release
```

Comment:
Unrelated to this PR, but we should we change the release version to 0.1.1?

Reply:
True, at least before we go for the release (either way we'll need to update it to `(current release)` later).